xml/System.IO.Compression/BrotliDecoder.xml

```diff
@@ -52,12 +52,12 @@
         <Parameter Name="bytesWritten" Type="System.Int32" RefType="out" />
       </Parameters>
       <Docs>
-        <param name="source">To be added.</param>
-        <param name="destination">To be added.</param>
-        <param name="bytesConsumed">To be added.</param>
-        <param name="bytesWritten">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="source">The byte read-only span containing the compressed data.</param>
+        <param name="destination">When this method returns, a byte span containing the decompressed data.</param>
```

Comment:
**Question:** Can the source/destination buffers overlap (or even be the same span)? Worth testing and adding a remark.

Reply:
The destination needs to be larger than the source, since it's a decompression task.
Also, I don't think we should document that since it's common sense to not decompress your input in the same span that contains your input.